kubernetes-ingress.el

```diff
@@ -0,0 +1,136 @@
+;;; kubernetes-ingress.el --- Rendering routines for Kubernetes ingress.  -*- lexical-binding: t; -*-
+;;; Commentary:
+;;; Code:
+
+(require 'dash)
+
+(require 'kubernetes-ast)
+(require 'kubernetes-loading-container)
+(require 'kubernetes-modes)
+(require 'kubernetes-props)
+(require 'kubernetes-state)
+(require 'kubernetes-utils)
+(require 'kubernetes-yaml)
+
+
+;; Components
+
+(defconst kubernetes-ingress--column-heading
+  (propertize (format "%-45s %6s" "Name" "Age")
```

Comment:
Can we add hosts, address and ports also?

Reply:
Added Hosts and Address. Ports was always 80. So ignored it.